megamek/src/megamek/server/Server.java

```diff
@@ -887,6 +887,16 @@ private void receivePlayerVersion(Packet packet, int connId) {
             System.out.println("ERROR: Client/Server Version Mismatch -- Client: "+version+" Server: "+MegaMek.VERSION);
             needs = true;
         }
+        else if (clientChecksum == null) {    // print a message indicating client doesn't have jar file
+            buf.append("Client Checksum is null. Client may not have a jar file");
+            System.out.println("ERROR: Client does not have a jar file");
+            needs = true;      	
+        }
+        else if (serverChecksum == null) {    // print message indicating server doesn't have jar file
```

Comment:
Same as line 890 -- this should go up a line to be on the same line as the closing bracket on line 894.